e2e/suite.go

```diff
@@ -16,14 +16,15 @@ import (
 	"syscall"
 	"testing"
 
-	"github.com/sylabs/singularity/e2e/actions"
```

Comment:
need to uncomment some lines around here and just add `"github.com/sylabs/singularity/e2e/key"` ?